### PR TITLE
feat(ImageSlider): Omniture tracking for feature companies

### PR DESCRIPTION
**Pull request by youwei91**
Refer to this card & PR:
https://trello.com/c/OvzMn5os/113-omniture-analytics-homepage-feature-tracking-for-featured-companies
https://github.com/seekinternational/srp-react/pull/595

To replace ahref with LinkComponent in ImageSlider & Omniture tracking feature companies

### CODE REVIEW

File: react/ImageSlider/ImageSlider.js [hunk 1]
```diff
@@ -85,6 +85,8 @@ export default class ImageSlider extends Component {
         </div>
         <div className={styles.containerDiv}>
           <Wrapper
+            onCompanyClick={onCompanyClick}
+            LinkComponent={LinkComponent}
```
Comment (PepeFranco) [inline react/ImageSlider/ImageSlider.js]: Could we change this prop name to something that tells us what this is used for?

Comment (youwei91) [reply react/ImageSlider/ImageSlider.js]: sure 👍

**Comment by mengtzu** [comment]
What's the rationale between the click and the link component being separate?  Couldn't you combine them in the consumer?

**Comment by youwei91** [comment]
@mengtzu the `onFeatureCompanyClick` is to trigger the omniture tracking, `LinkComponent` is to replace the ahref for better handling

**Comment by mengtzu** [comment]
@youwei91 I understand that previous uses of this pattern have had `LinkRenderer` handle both roles.

**Comment by youwei91** [comment]
I see...I was looking at `JobCard` as example, didn't notice `LinkRenderer` pattern exist 🤦‍♂️

**Pull request merged**